mozilla_vpn/views/subscriptions.view.lkml

```diff
@@ -190,33 +181,6 @@ view: +subscriptions {
     sql: GENERATE_DATE_ARRAY(${subscription_start_date}, ${end_date} - 1);;
   }
 
-  dimension: events {
-    hidden: yes
-    sql:
-      ARRAY_CONCAT(
-        [
-          STRUCT(
-            ${subscription_start_date} AS date,
-            "New" AS type,
-            IF(${subscription_start_date} = ${customer_start_date}, "New", "Resurrected") AS granular_type,
-            1 AS delta
-          )
-        ],
-        IF(
-          ${end_date} < DATE(${metadata.last_modified_date}),
-          [
-            STRUCT(
-              ${end_date} AS date,
-              "Cancelled" AS type,
-              ${cancel_reason} AS granular_type,
-              -1 AS delta
-            )
-          ],
-          []
-        )
-      );;
-  }
```

Comment:
If there's a need for having granular events at the individual subscription level we could potentially add that to the ETL.

Reply:
I am not aware that we have this need yet.  We can discuss when the need arises.  Thanks.